packages/vscode-extension/package.json

```diff
@@ -452,9 +452,11 @@
     "typecheck": "tsc --noEmit",
     "format": "prettier --write --list-different src",
     "build:tests": "tsc --project tsconfig.test.json",
-    "test": "npm run build:extension-code && npm run build:webview && npm run build:tests && vscode-test"
+    "test": "npm run build:extension-code && npm run build:webview && npm run build:tests && vscode-test",
+    "preinstall": "cd ../vscode-js-debug && git submodule update && npm install && npx tsc"
```

Comment:
we shouldn't use `npx` in scripts like that, this may result in different versions of tsx being run for different people. I'd assume that repo already has tsc as dev-dependency in which case it shouldn't require `npx` to run it

Reply:
Changed in: https://github.com/software-mansion/radon-ide/pull/846/commits/21e4c4f7ccd808540321a849248f272eea00ef3b